src/auth/auth-config.ts

```diff
@@ -506,6 +507,28 @@ export interface MultiFactorConfig {
    * Currently only ‘phone’ is supported.
    */
   factorIds?: AuthFactorType[];
+  /**
+   * A list of multi-factor provider specific config. 
+   * New MFA providers (except phone) will indicate enablement/disablement through this field.
+   */
+  providerConfigs?: MultiFactorProviderConfig[];
+}
+
+export interface MultiFactorProviderConfig {
```

Comment:
Let's add documentation to this and all the new types below:

example see: `MultiFactorConfig`
```
/**
 * Interface representing a multi-factor configuration.
 * This can be used to define whether multi-factor authentication is enabled
 * or disabled and the list of second factor challenges that are supported.
 */
 ```